lib/Doctrine/Migrations/DefaultParameterFormatter.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Types\Type;
+use function array_map;
+use function implode;
+use function is_array;
+use function is_bool;
+use function is_int;
+use function is_string;
+use function sprintf;
+
+/**
+ * The DefaultParameterFormatter class is responsible for formatting SQL query parameters to a string
+ * for display output.
+ *
+ * @internal
+ */
+final class DefaultParameterFormatter implements ParameterFormatter
```

Comment:
I read you were not happy with that prefix… may I suggest `InlineParameterFormatter` here? Since it produces one line with all the parameters, as opposed to a table? Should  the test class be renamed too?

Reply:
Good idea! thanks!